app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -87,7 +85,7 @@ object FeatureFlags {
     /**
      * Enables showing the wallpaper functionality.
      */
-    val showWallpapers = Config.channel.isNightlyOrDebug
+    const val showWallpapers = true
```

Comment:
Do we need this flag at all if it's true?

Reply:
A guess for the moment, it help us to easily turn it off/on if needed it 😅.  Until we feel the feature it's completely mature.

Reply:
When I add a feature flag, I file a ticket that is linked in the comment of the flag to evaluate when to remove the flag.

For example, this is the issue I used for WebAuthn: https://github.com/mozilla-mobile/fenix/issues/19785

Reply:
👍🏽

Reply:
I filed https://github.com/mozilla-mobile/fenix/issues/24050 for it, thanks @jonalmeida!